mit_frontend/src/routers/ForgetPassword.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import {Card, CardTitle} from 'material-ui/Card';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../App.css';
+import styles from './styles';
+
+class ForgetPassword extends Component {
```

Comment:
Will you implement sending password reset link through email? Or it will be Joseph's work?